src/main/java/dook/services/UiDisplay.java

```diff
@@ -0,0 +1,64 @@
+package dook.services;
+
+import java.util.Scanner;
+
+import dook.command.CommandInfo;
+
+/**
+ * Responsible for displaying messages to the user, as well as reading user commands.
+ */
+public class UiDisplay {
```

Comment:
Good job making sure UI is written as Ui for the class name, as abbreviations should not be in upper case when part of a name